repo2docker/buildpacks/python/__init__.py

```diff
@@ -238,19 +247,21 @@ def get_build_scripts(self):
     def get_assemble_scripts(self):
         """Return series of build-steps specific to this repository.
         """
-        return super().get_assemble_scripts() + [
-            (
+        requirements_txt = self.binder_path('requirements.txt')
+        assemble_scripts = super().get_assemble_scripts()
+        if os.path.exists(requirements_txt):
+            assemble_scripts.append((
                 '${NB_USER}',
-                'pip2 install --no-cache-dir -r requirements.txt'
-            )
-        ]
+                'pip3 install --no-cache-dir -r "{}"'.format(requirements_txt)
```

Comment:
I think this should be `pip2` and not `pip3`. If the change is on purpose we should leave a comment because it is confusing to have `pip3` in the Python 2 build pack.